addons/web/static/src/core/file_upload/file_upload_service.js

```diff
@@ -24,7 +24,7 @@ export const fileUploadService = {
 
         /**
          * @param {string}                          route
-         * @param {FileList}                        files
+         * @param {FileList|Array<File>}           files
```

Comment:
![image](https://user-images.githubusercontent.com/41282536/229515932-5e76ad55-1617-4926-8277-44d196a1b90d.png)

@pyduf Parameter is no longer visually aligned with the others :eyes:

Reply:
Thanks, it's updated.